overlord/servicestate/quota_handlers.go

```diff
@@ -476,7 +478,7 @@ func ensureSnapServicesForGroup(st *state.State, t *state.Task, grp *quota.Group
 		}
 
 		st.Unlock()
-		err = wrappers.RestartServices(startupOrdered, nil, nil, meter, perfTimings)
+		err = wrappers.RestartServices(startupOrdered, nil, nil, meterUnlocked, perfTimings)
```

Comment:
Does it mean we had a bug here passing same meter everywhere?

Reply:
Yeah but the meter was not used at all, it will be used in https://github.com/snapcore/snapd/pull/10420 however